src/I18n/DateFormatTrait.php

```diff
@@ -42,6 +42,15 @@ trait DateFormatTrait
      */
     protected static $defaultLocale;
 
+    /**
+     * Whether the IntlDateFormatter is set to lenient for parsing.
+     *
+     * Defaults to null which uses IntlDateFormatter default (true).
+     *
+     * @var bool|null
+     */
+    protected static $lentientParsing;
```

Comment:
Why don't you just use true/false instead of null/false?

Reply:
Using two falsey values to indicate different states is not intuitive.

Reply:
I was trying not to hide the intldateformatter default but that's too complicated for this.